app/views/layouts/simulator.html.erb

```diff
@@ -4,8 +4,10 @@
     <%= csrf_meta_tags %>
     <script>window.locale = "<%= I18n.locale %>"</script>
     <script src="/js/pace.js"></script>
-    <%= stylesheet_link_tag "simulator", media: "all", "data-turbolinks-track": "reload" %>
-    <%= javascript_include_tag "simulator", "data-turbolinks-track": "reload" %>
+    <script src="/js/vendor/jquery.min.js"></script>
+    <script src="/js/vendor/bootstrap.bundle.min.js"></script>
```

Comment:
Do not use vendored libraries as it will be difficult to update, Just use the Jquery version from the npm and bundle that using Vite, https://github.com/vitejs/vite/discussions/3744

Reply:
Ok. 
Will fix it

Reply:
@tachyons now it's fixed